src/locales/en_US.js

```diff
@@ -705,6 +705,8 @@ const strings = {
   string_no_wallet_name: 'No Wallet Name',
   export_transaction_error: 'Start date should be earlier than the end date',
   export_transaction_export_error: 'No transactions in the date range chosen',
+  export_transaction_export_error_2: 'Select only one export type on android',
```

Comment:
This string should not be needed. On Android force the user to not set more than one export type.